### PR TITLE
Feat/remove uiwebview itms 90809

### PR DESCRIPTION
**Pull request by m-abs**
Removes UIWebView as Apple don't want developers to use it anymore

Implements: #49.

BREAKING CHANGES:
UIWebView support is removed, this mean x-local:// won't work for iOS < 11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,22 +50,18 @@ This registers the element `WebViewExt`. Replace the <WebView> tag with <WebView
...
 
 ## API
 
 ### NativeScript View
 
 | Property | Value | Description |
 | --- | --- | --- |
-| readonly isUIWebView | true / false | Is the native webview an UIWebView? True if `iOS <11` |
-| readonly isWkWebView | true / false | Is the native webview an WKWebView? True if `iOS >=11` |
+| readonly supportXLocalSchema | true / false | Is `x-local://` supported? True on `iOS >= 11` or `Android`, False on `iOS < 11`. |
```
Comment (ddfreiling) [inline README.md]: *scheme

File: README.md [hunk 2]
```diff
@@ -105,10 +100,10 @@ The custom `NSURLProtocol` used with UIWebView is shared with all instances of t
 | loadStarted | Raised when a loadStarted event occurs. args is a `LoadStartedEventData` |
 | shouldOverrideUrlLoading | Raised before the webview requests an URL. Can cancelled by setting args.cancel = true in the `ShouldOverrideUrlLoadEventData` |
 | titleChanged | Document title changed |
-| webAlert | Raised when `window.alert` is triggered inside the webview, needed to use customs dialogs for web alerts. args in a `WebAlertEventData`. `args.callback()` must be called to indicate alert is closed. **NOTE:** Not supported by UIWebView |
-| webConfirm | Raised when `window.confirm` is triggered inside the webview, needed to use customs dialogs for web confirm boxes. args in a `webConfirmEvent`. `args.callback(boolean)` must be called to indicate confirm box is closed. **NOTE:** Not supported by UIWebView |
+| webAlert | Raised when `window.alert` is triggered inside the webview, needed to use customs dialogs for web alerts. args in a `WebAlertEventData`. `args.callback()` must be called to indicate alert is closed. |
```
Comment (ddfreiling) [inline README.md]: custom* dialogs

File: src/webview-ext-common.ts [hunk 3]
```diff
@@ -345,9 +339,8 @@ export class WebViewExtBase extends ContainerView {
 
     /**
      * Gets the native [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview/) that represents the user interface for this component. Valid only when running on iOS 11+.
-     * Gets the native [UIWebView]https://developer.apple.com/documentation/uikit/uiwebview that represents the user interface for this component. Valid only when running on iOS <11
      */
-    public ios: any /* WKWebView | UIWebView */;
+    public ios: any /* WKWebView */;
```
Comment (ddfreiling) [inline src/webview-ext-common.ts]: could just give it the WKWebView type now.

Comment (m-abs) [reply src/webview-ext-common.ts]: Native type is usually not exported, due to the size of the typings needed.

Comment (ddfreiling) [reply src/webview-ext-common.ts]: 👍

**Pull request merged**